docs/source/markdown/podman-create.1.md

```diff
@@ -870,6 +874,23 @@ where source dir is mounted on) has to have right propagation properties. For
 shared volumes, source mount point has to be shared. And for slave volumes,
 source mount has to be either shared or slave.
 
+If you want to recursively mount a volume and all of it's submounts into a
+container, then you can use the `rbind` option.  By default --volumes use the
```

Comment:
Suggest "By default the `bind` option is used, and submounts of the source directory will not be mounted into the container"

Reply:
Fixed